.travis.yml

```diff
@@ -0,0 +1,23 @@
+sudo: required
+
+language: python
+
+services:
+  - docker
+
+env:
+  matrix:
+    - ODOO_VERSION="10.0"
+    - ODOO_VERSION="9.0"
+
+before_install:
+  - docker pull kiasaki/alpine-postgres
+  - docker run -d -e POSTGRES_USER=odoo -e POSTGRES_PASSWORD=odoo --name db kiasaki/alpine-postgres
+
+install:
+  - docker build -t $TRAVIS_BUILD_NUMBER --build-arg ODOO_VERSION=$ODOO_VERSION $TRAVIS_BUILD_DIR/
+
+script:
+  - docker run -d -p 8069:8069 --name $TRAVIS_BUILD_NUMBER --link db:db -t $TRAVIS_BUILD_NUMBER
+  - sleep 10
+  - curl --fail http://localhost:8069/web/database/manager
```

Comment:
I considered that, but typically Travis  is a log of everything. What if it fails, for example?